### ISSUE TITLE
twoCharLinebreak doesn't work in Firefox

### ISSUE THREAD

**Issue by JamesChe91** [mimo84/bootstrap-maxlength#113]
How I can fix this?

**Comment by JamesChe91** [comment]
Look a screenshot
https://k60.kn3.net/2/3/F/C/8/B/0A1.png
The number of characters entered exceeds the limit when there are line breaks.

**Comment by reypm** [comment]
yes, confirmed, I am running the same issue: Chrome 48.0.2564.97 m, jQuery 2.2.0, Bootstrap 3.3.6 and latest Bootstrap Maxlength release from master branch. This is my setup:

    $('#message').maxlength({
        alwaysShow: true,
        threshold: 10,
        warningClass: "label label-info",
        limitReachedClass: "label label-danger",
        placement: 'top',
        utf8: true
    });

Any advice @mimo84?  

Also I am using this for check how many characters a message has since this will be send through SMS and the limit is 160 for new phones and 140 for a few old phones (don't ask me why) so is there any chance to alert user when I reach 140 characters?

**Comment by mimo84** [comment]
I didn't have time to debug this bug unfortunately.. I'll try to look into it as soon as I have time. In the meanwhile do you have any ideas why this should have stopped working?

**Comment by reypm** [comment]
@mimo84 well I think you miss a case when we mix special characters like in spanish languages ex: áéíóúñ or any other with line breaks. For example I am testing with this text:

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam sed ipsum odio. Ut vel semper odio. Morbi eu urna ullamcorper, facilisis leo id

áéíóú ñ
Lore
```

And it allows 166 over 160. [Here][1] is a fiddle with the tests. Hope you have a time for look on this soon since I check the source but didn't found really where it fails

 [1]: https://jsfiddle.net/Guruprasad_Rao/fexqf0vu/10/

**Comment by ashleyglee** [comment]
This is a browser issue on how different browsers count newlines.  Chrome counts them as 2 characters, while IE/Firefox count them as 1 character.  The default setting is "twoCharLinebreak: true" which is why you are seeing linebreaks counted as 2 characters in IE/Firefox.  If you want the library to also prevent you entering over that limit, set the "validate" option to true, that's by false by default:
```
$('#message').maxlength({
    validate: true,  // set this for this library to enforce maxlength
    alwaysShow: true,
    threshold: 10,
    warningClass: "label label-info",
    limitReachedClass: "label label-danger",
    placement: 'bottom',
    utf8: true
});
```
